### PR TITLE
Feature/toggle button

### PR DESCRIPTION
**Pull request by capetillo**
### DESCRIPTION

**Background:**
Part of DataLab's mockup presentation included the ability for users to toggle views when they're on the `ProjectView` so that they can choose between an image carousel display or a list (OCS inspired)

**Implementation:**
To the already existing `ToggleButton` component, I first declared a reactive constant that's set to `true` and then added logic to change its value from true to false and vice versa. This last piece gets invoked when the button is clicked. And then I conditionally rendered based on what's selected.

### VISUALS
**Video of display changing on click of toggle button**

https://github.com/LCOGT/datalab-ui/assets/54489472/f6f726e3-e0d6-499c-8ad9-f367be78c578

### CODE REVIEW

File: src/components/ToggleButton.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+</script>
+
+<template>
+        
+    <v-btn @click="toggleView">
+        <div v-if="isCarouselView">List Display</div>
+        <div v-else> Image Display </div>
+    </v-btn>
+  
+
```
Comment (mgdaily) [inline src/components/ToggleButton.vue:21]: need this extra whitespace?

File: src/components/ProjectView/ProjectImages.vue [hunk 2]
```diff
@@ -16,10 +15,7 @@
...
 <template>
-    <v-card>
         <!-- These divs are just for demonstration purposes, feel free to remove them when implemented -->
-        <div>{{ selectedProject }}</div>
-        <image-list :data="MockData" :selectedImages="selectedImages" @update-selected="updateSelectedImages"/>
-        <ImageCarousel></ImageCarousel>
-    </v-card>
+    <div>{{ selectedProject }}</div>
+    <image-list :data="MockData" :selectedImages="selectedImages" @update-selected="updateSelectedImages"/>
 </template>
```
Comment (mgdaily) [inline src/components/ProjectView/ProjectImages.vue:21]: Missing final newline - VSCode will do this for you! https://stackoverflow.com/a/44704969

File: src/components/ToggleButton.vue [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    <div v-if="isCarouselView">
+    <ImageCarousel/>
+  </div>
+
+  <div v-else>
+    <ProjectImages/>
+  </div>
+</template>
```
Comment (mgdaily) [inline src/components/ToggleButton.vue:30]: final newline!

File: src/views/ProjectView.vue [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
+    import ToggleButton from '../components/ToggleButton.vue';
     import ImportButton from '../components/ImportButton.vue';
 </script>
 
 <template>
-    <project-list/>
-    <import-button/>
+    <ToggleButton />
+    <ImportButton/>
 </template>
```
Comment (mgdaily) [inline src/views/ProjectView.vue:9]: Final newline yadda yadda

**Review by mgdaily — APPROVED** [review]
Approved, pending a few fixes! Video looks cleeeaaaan

File: src/components/ToggleButton.vue (hunk 1 above)
Comment (capetillo) [reply src/components/ToggleButton.vue:21]: no, classic me

**Pull request merged**